website/docs/r/rds_cluster.html.markdown

```diff
@@ -190,6 +190,30 @@ resource "aws_rds_cluster" "example" {
 * `seconds_until_auto_pause` - (Optional) The time, in seconds, before an Aurora DB cluster in serverless mode is paused. Valid values are `300` through `86400`. Defaults to `300`.
 * `timeout_action` - (Optional) The action to take when the timeout is reached. Valid values: `ForceApplyCapacityChange`, `RollbackCapacityChange`. Defaults to `RollbackCapacityChange`. See [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.how-it-works.html#aurora-serverless.how-it-works.timeout-action).
 
+### restore_to_point_in_time Argument Reference
+
+Example:
+
+```hcl
+resource "aws_rds_cluster" "example-clone" {
+  # ... other configuration ...
+
+  restore_To_point_in_time {
```

Comment:
```suggestion
  restore_to_point_in_time {
```

Reply:
I would also consider moving this entire block to below the `S3 Options` section to keep it close to that related section